### DISCUSSION TITLE
Awaken dont worked

### DISCUSSION THREAD

**Discussion by SnusM** [SnosMe/awakened-poe-trade#1151] [Q&A]
Here comes a new league and I come with this problem for the second time, when opening the program, not a single button combination works, although the application itself is open and working, in the last league I found a way to solve this problem by downloading the old version of the awakener and setting it up through the settings, but now I don't remember how I did it, yes and I don't like the prospect of returning here every league either :(

**Comment by SnosMe** [comment]
try removing config

**Reply by SnusM** [reply]
After deleting the config, nothing happened, if I understood correctly, it is located by the user-AppData-Roaming-AWP-app data and there I deleted the files

**Reply by SnusM** [reply]
I looked through the forum and found something similar, in the program itself you need to change the default settings to the league, but how to do it when shift + space does not work

**Reply by SnusM** [reply]
I solved the problem, the antivirus was interfering
